composed_api.rst

```diff
@@ -287,20 +276,20 @@ Endpoint
 
 ::
 
-    GET <instance_url>/<instance_root>_ba/api/v1/aoi/add
+    GET <instance_url>/<instance_root>_ba/api/v2/aoi/add
 
 Request Parameters
 ^^^^^^^^^^^^^^^^^^
 
 +-------------------+-------------------------------------------------------+
 | Query parameter   | Value                                                 |
 +===================+=======================================================+
+| source            | *Required*. Your GRiD generated API key.              |
```

Comment:
I think tracking the application is fine. I just don't think this does that for you. Unless I'm overlooking something. If GQ were updated to v1 or v2, the user would have to log onto GRiD to create that key (by whatever means). I can't just hand them a key that is somehow linked to GQ, and now part of creating the key indicates that it will be used by GQ. It could be used by any app to talk to the API.